src/catch2/reporters/catch_reporter_json.cpp

```diff
@@ -198,6 +211,12 @@ namespace Catch {
         }
     }
     void JsonReporter::listTests( std::vector<TestCaseHandle> const& tests ) {
+        if ( !m_startedListing ) {
+            startListing();
+            m_startedListing = true;
+        }
+
+        // TODO: Why? This should just assert. If the object is not there, there is a bug.
```

Comment:
I wasn't sure what to do when writing this, since I saw something similar to this in the XML reporter, I followed it